docker/webui/docker-compose-example.yaml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+services:
+  data:
+    image: openqa_data
+    restart: always
+    volumes:
+      - ./data/factory:/data/factory
+      - ./data/tests:/data/tests
+  backend:
```

Comment:
It doesn't look like this is about the "backend". My main review comment contains more details what I mean by that.